subsystems/provisioning/provisioning.go

```diff
@@ -63,9 +64,23 @@ type Provisioning struct {
 	grpcServer *grpc.Server
 	portalData *portalData
 
+	// Toggle for bluetooth/WiFi provisioning method
+	muProvisioningMethod sync.Mutex
+	provisioningMethod   provisioningMethod
+
+	// BLE peripheral used for bluetooth provisioning (if selected as desired provisoning method).
+	blePeripheral blep.BLEPeripheral
+
 	pb.UnimplementedProvisioningServiceServer
 }
 
+type provisioningMethod string
+
+const (
+	provisioningMethodHotspot provisioningMethod = "WiFi"
+	provisioningMethodBLE     provisioningMethod = "BLE"
```

Comment:
Let's call this `bluetooth`. For users, setting the configuration it will be more obvious what bluetooth means.